config/routes/Centreon/security/authentication.yaml

```diff
@@ -4,9 +4,9 @@ centreon_security_authentication_find_providers_configurations:
     controller: 'Centreon\Application\Controller\AuthenticationController::findProvidersConfigurations'
 
 centreon_security_authentication_login:
-    methods: GET|POST
+    methods: POST
     path: /authentication/providers/configurations/{providerConfigurationName}
```

Comment:
check requirement for `providerConfigurationName`

Reply:
regarding documentation, requirements seems only useful in specific case like route collision : https://symfony.com/doc/current/routing.html#parameters-validation
Then, I don't add it here